packages/formation/sass/modules/_m-action-link.scss

```diff
@@ -3,14 +3,15 @@ a.vads-c-action-link--blue, a.vads-c-action-link--green, a.vads-c-action-link--w
     font-family: "Font Awesome 5 Free";
     font-weight: 900;
     font-size: 175%;
-    display: inline-block; // Remove the underline text decoration
     content: "\f138";
     padding-right: 1rem;
-    transform: translateY(5px); // "Center" the circle with the text
+    transform: translateY(-9px); // "Center" the circle with the text
     height: 0px; // So the height doesn't mess with the focus halo
+    float: left;
```

Comment:
Can this be better achieved using flexbox? There was a similar issue with radio button labels that was solved that way: https://github.com/department-of-veterans-affairs/component-library/pull/633

Reply:
Thanks for explaining and for the screenshot!  I can see what you mean now. Without making structural changes, I'm not sure this can be accomplished another way either (I'm not sure why `text-decoration: none` isn't working in this case).